lib/rubocop/cop/mixin/preceding_following_alignment.rb

```diff
@@ -89,6 +101,81 @@ def aligned_assignment?(range, line)
       def aligned_identical?(range, line)
         range.source == line[range.column, range.size]
       end
+
+      def aligned_with_assignment(token, line_range)
+        token_line_indent    = processed_source
+                               .line_indentation(token.line)
+        assignment_lines     = relevant_assignment_lines(line_range)
+        relevant_line_number = assignment_lines[1]
+
+        return :none unless relevant_line_number
```

Comment:
`:none` is perhaps too similar to `:no`, so it could be confusing for the reader. I'd prefer `:not_found`.

Reply:
@jonas054 `:none` is used in other similar cops as well for a similar meaning.  If I change it, I'd prefer to change it everywhere.

Examples:

* https://github.com/rubocop-hq/rubocop/blob/26e7b08d319a441a02937dae46db583b01561db3/lib/rubocop/cop/mixin/surrounding_space.rb#L81
* https://github.com/rubocop-hq/rubocop/blob/26e7b08d319a441a02937dae46db583b01561db3/lib/rubocop/cop/lint/unneeded_cop_enable_directive.rb#L89

Reply:
Oh, I didn't realize. Better leave it as-is then.